x/batching/keeper/querier.go

```diff
@@ -42,14 +52,23 @@ func (q Querier) BatchForHeight(c context.Context, req *types.QueryBatchForHeigh
 	}, nil
 }
 
-func (q Querier) Batches(c context.Context, _ *types.QueryBatchesRequest) (*types.QueryBatchesResponse, error) {
+func (q Querier) Batches(c context.Context, req *types.QueryBatchesRequest) (*types.QueryBatchesResponse, error) {
```

Comment:
Should we not add validation on the `req.Pagination` size/limit to check that it doesn't exceed some reasonable maximum? I looked briefly at `query.CollectionPaginat` but didn't see any checks there.